src/App.css

```diff
@@ -1,4 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Pacifico&display=swap');
+
 .App {
-  box-sizing: border-box;
-  text-align: center;
+	box-sizing: border-box;
+	text-align: center;
+	margin: 0;
+}
+
+.no-videos {
+	padding: 2rem;
+	font-size: 1.25rem;
+	color: var(--midgrey);
+}
+
+.width-100 {
+	width: 100%;
+}
+
+.heading {
+	/* width: 100%; */
```

Comment:
minor suggestion: commented code can be removed

Reply:
Removed the comments